### PR TITLE
Feature/argodev 512

### PR DESCRIPTION
**Pull request by vidkris**
Feature/argodev 512: Adding the code for the case 201_202_202

### CODE REVIEW

File: bgc_derivation/processo2.m [hunk 1]
```diff
@@ -15,6 +15,7 @@ function processo2(floatnos,varargin)
 %      Also no wiring yet for certificate values
 
 %  title - s processo2  vr - 1.0  authot - bodc/sgl  date - 20211112
```
Comment (qtl-bodc) [inline bgc_derivation/processo2.m:17]: spelling 'authot'

File: bgc_derivation/processo2.m [hunk 2]
```diff
@@ -15,6 +15,7 @@ function processo2(floatnos,varargin)
...
+%                       vr -1.0.1 authot -bodc/vidkri date - 20020816 added the case 201_202_202
```
Comment (qtl-bodc) [inline bgc_derivation/processo2.m:18]: spelling 'authot'

File: bgc_derivation/processo2.m [hunk 3]
```diff
@@ -135,8 +138,36 @@ function processo2(floatnos,varargin)
...
+          T  =  ncctd{'TEMP'}(:);
+          S  =  ncctd{'PSAL'}(:);
+          P  =  ncctd{'PRES'}(:);
+          bphasedoxy=ncdox{'BPHASE_DOXY'}(:);
+          rphasedoxy=ncdox{'RPHASE_DOXY'}(:);
+          doxy  =  ncdox{'DOXY'}(:);
+          
+          %retrieving the calibration coeffs.
+          stc  =  stcoeff(1).optode2; % could be many optodes.sgl working on geto2sensor to solve the issue.         
+          maskctdS = S == 99999; % as defined in the .nc file
```
Comment (qtl-bodc) [inline bgc_derivation/processo2.m:152]: maskctdS and mask not used anymore, perhaps comment or delete.

**Review by qtl-bodc — COMMENTED** [review]
Not sure if we need to add the version changes explicitly in the header comment as all this should be captured and managed in GitHub?

File: bgc_derivation/processo2.m (hunk 2 above)
Comment (vidkris) [reply bgc_derivation/processo2.m:18]: removed this line as it will be captured in github

File: bgc_derivation/processo2.m (hunk 1 above)
Comment (vidkris) [reply bgc_derivation/processo2.m:17]: done

File: bgc_derivation/processo2.m (hunk 3 above)
Comment (vidkris) [reply bgc_derivation/processo2.m:152]: I used the mask for just the salinity. If you think that is not the right approach, I will remove that.

Comment (qtl-bodc) [reply bgc_derivation/processo2.m:151]: I mean that the maskctdS and mask variable doesn't seem to be used further down the code, so if redundant then probably the mastkctdS and mask variable can be deleted.

**Review by qtl-bodc — APPROVED** [review]

**Pull request merged**